NebulaModel/Packets/Factory/Belt/BeltReverseRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace NebulaModel.Packets.Factory.Belt;
+
+public class BeltReverseRequest
+{
+    public BeltReverseRequest() { }
+
+    public BeltReverseRequest(int beltId, int planetId, int authorId)
```

Comment:
This should be `BeltReverseRequestPacket` (the filename as well) to fit in with the rest of the files in that section